mParticle-Apple-SDK/mParticle.m

```diff
@@ -159,6 +159,89 @@ - (NSString *)description {
 @implementation MPDataPlanOptions
 @end
 
+@implementation MPRokt
+
+- (void)selectPlacements:(NSString * _Nullable)identifier
+              attributes:(NSDictionary<NSString *, NSString *> * _Nullable)attributes {
```

Comment:
ditto to above comment

Reply:
i actually think there's a stronger case for nil here because if you're using mParticle you basically don't need to pass attributes in. Assuming you've been adding them as user attributes